### PR TITLE
qml: fixed receive wallet display + disable send/receive without wallets

### PR DESCRIPTION
**Pull request by dvajdual**

### CODE REVIEW

File: GUI/QtQuick/qml/Receive/ReceiveQrCode.qml [hunk 1]
```diff
@@ -43,11 +44,20 @@ ColumnLayout  {
...
         font.pixelSize: 14
         font.family: "Roboto"
         font.weight: Font.Normal
+
+        Connections
+        {
+            target:walletBalances
+            function onRowCountChanged ()
+            {
+                wallet_name.text = getWalletData(overviewWalletIndex, WalletBalance.NameRole) + " / Native SegWit"
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Receive/ReceiveQrCode.qml]: Can't we avoid code duplication here and above? Use one function in both places. Better to have it as a translatable string: `qsTr("%1 / Native SegWit").arg(<wallet name>)`

File: GUI/QtQuick/qml/main.qml [hunk 2]
```diff
@@ -198,13 +198,16 @@ ApplicationWindow {
...
                 Layout.fillHeight: true
                 onClicked: {
-                    bsApp.requestFeeSuggestions()
-                    topMenuBtnClicked(btnSend)
-                    //stack.push(sendPage)
-                    send_popup.init()
-                    send_popup.show()
-                    send_popup.raise()
-                    send_popup.requestActivate()
+                    if (walletBalances.rowCount > 0)
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/main.qml:207]: Corresponding buttons should be disabled, too, I guess?

Comment (dvajdual) [reply GUI/QtQuick/qml/main.qml:207]: Dont sure, maybe we will add message box if no wallets - "Please add at least 1 wallet to can sending bitcoins"

File: GUI/QtQuick/qml/Receive/ReceiveQrCode.qml (hunk 1 above)
Comment (dvajdual) [reply GUI/QtQuick/qml/Receive/ReceiveQrCode.qml]: Hmm qsTr need only to Native Segwit. We will not translate wallet name I think)

File: GUI/QtQuick/qml/main.qml (hunk 2 above)
Comment (sergey-chernikov) [reply GUI/QtQuick/qml/main.qml:207]: Let's just disable for now, or redirect to wallet create dialog.

File: GUI/QtQuick/qml/Receive/ReceiveQrCode.qml (hunk 1 above)
Comment (sergey-chernikov) [reply GUI/QtQuick/qml/Receive/ReceiveQrCode.qml]: Yes, this is obvious if you look at my first comment here. Change to the suggested format please.

File: GUI/QtQuick/qml/Receive/ReceiveQrCode.qml [hunk 3]
```diff
@@ -43,11 +44,25 @@ ColumnLayout  {
         Layout.alignment: Qt.AlignCenter
         Layout.topMargin: 6
         Layout.preferredHeight : 16
-        text: bsApp.walletsList[overviewWalletIndex] + " / Native SegWit"
+        text:  fill_in_text()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Receive/ReceiveQrCode.qml]: `fill_in_text()` should return a value, I guess?

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**